examples/CMakeLists.txt

```diff
@@ -50,7 +50,13 @@ set_target_properties(StaticJxlThreads PROPERTIES
 add_executable(decode_oneshot_static decode_oneshot.cc)
 target_link_libraries(decode_oneshot_static
   -static StaticJxl StaticJxlThreads)
+if (NOT JPEGXL_ENABLE_SKCMS)
+  target_link_libraries(decode_oneshot_static -static lcms2)
```

Comment:
This `-static` should be unnecessary since it’s already above. It doesn’t specifically affect the linking of lcms2 as far as CMake is concerned, it just adds `-static` to the linker flags.